contrib/azure/arm-template.json

```diff
@@ -87,6 +87,13 @@
       "metadata": {
         "description": "Version of CoreOS to deploy."
       }
+    },
+    "storageAccountType": {
+      "type": "string",
+      "metadata": {
+        "description": "Storage account type. Can be Premium_LRS (recommended) or Standard_LRS"
+      },
+      "defaultValue": "Premium_LRS"
```

Comment:
Can you add `allowedValues` (see https://azure.microsoft.com/en-us/documentation/articles/resource-group-authoring-templates/#parameters) so there is little room for mistakes.

Not sure how the view for this looks but would probably reduce the `description` part.

You may also want to add the rest of the ones from https://azure.microsoft.com/en-us/documentation/articles/resource-manager-template-storage/ - it's mostly just geo redundant and that like but won't hurt anything.